front_end/src/app/(main)/questions/[id]/components/forecast_maker/forecast_maker_conditional/forecast_maker_conditional_binary.tsx

```diff
@@ -50,14 +53,33 @@ const ForecastMakerConditionalBinary: FC<Props> = ({
   const { user } = useAuth();
   const { hideCP } = useHideCP();
 
-  const prevYesForecastValue = extractPrevBinaryForecastValue(prevYesForecast);
-  const prevNoForecastValue = extractPrevBinaryForecastValue(prevNoForecast);
-  const hasUserForecast = !!prevYesForecastValue || !!prevNoForecastValue;
-
   const { condition, condition_child, question_yes, question_no } = conditional;
   const questionYesId = question_yes.id;
   const questionNoId = question_no.id;
 
+  const latestYes = question_yes.my_forecasts?.latest;
+  const latestNo = question_no.my_forecasts?.latest;
+  const prevYesForecastValue =
+    latestYes && !latestYes.end_time
+      ? Math.round(latestYes.forecast_values[1] * 10000) / 100
+      : null;
+  const prevNoForecastValue =
+    latestNo && !latestNo.end_time
+      ? Math.round(latestNo.forecast_values[1] * 10000) / 100
+      : null;
```

Comment:
Q: We can't use `extractPrevBinaryForecastValue` anymore? 
Also, looks like `prevYesForecast` and `prevNoForecast` are not used anymore (actually used for sending GA events, but I assume it should be replaced with `prevYesForecastValue` and `prevNoForecastValue`)